plugins/omelasticsearch/omelasticsearch.c

```diff
@@ -127,8 +133,9 @@ typedef struct wrkrInstanceData {
 /* tables for interfacing with the v6 config system */
 /* action (instance) parameters */
 static struct cnfparamdescr actpdescr[] = {
-	{ "server", eCmdHdlrGetWord, 0 },
+	{ "server", eCmdHdlrArray, 0 },
```

Comment:
The existing tests rely on the default. As far as the module is concerned, the two code paths are 1- array parameter or 2- default. 
Since the eCmdHdlrArray guarantees an array, I didn't create a test with the server specified as just a string, but i can add one in if you think it's worth it.